user-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/user/rest/HealthActuatorContractTest.java

```diff
@@ -33,8 +34,12 @@
  * This is done by checking the service is still alive and healthy by REST GET /actuator/health.
  * This should return 200 OK if the service is healthy.
  */
-@SpringBootTest(classes = UserApplication.class, webEnvironment = WebEnvironment.RANDOM_PORT)
-@ActiveProfiles({"caffeine", "akkatest"})
+@SpringBootTest(
+        classes = UserApplication.class,
+        webEnvironment = WebEnvironment.RANDOM_PORT,
+        properties = {"management.health.kafka.enabled=false", "spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.data.redis.RedisAutoConfiguration"}
```

Comment:
This Redis exclusion shouldn't be required if using the caffeine profile

Reply:
Removed the exclusions